sssd_test_framework/roles/client.py

```diff
@@ -94,3 +94,18 @@ def setup(self) -> None:
                     raise ValueError("List is not expected")
 
                 self.sssd.import_domain(domain, role)
+
+    def set_sssd_user(self, user: str) -> bool:
+        """
+        Set [sssd]/user option.
+
+        :param user: Option value to set.
+        :type user: str
+        :return: True if success, false otherwise
+        :rtype: bool
```

Comment:
Wouldn't it make more sense to raise an error instead of returning bool?

Reply:
Done.